### ISSUE TITLE
Does this plugin support ddl export？

### ISSUE THREAD

**Issue by andonr** [mtxr/vscode-sqltools#486]
**Is your feature request related to a problem? Please describe.**
I want to export table create sql(DDL),but  I can NOT find the menu.

**Describe the solution you'd like**
When I right click the table,there is a menu which I can click to show the ddl of the table.

**Comment by Cody-at-Benjamin** [comment]
+1

**Comment by oleksii-nikitchuk** [comment]
yeah, this is one of the blockers for me to move away from other IDE.
It would be great to see table definitions similarly as you can see class or function definitions
